cmake-tool/helpers/tls_rootserver.lds

```diff
@@ -12,7 +12,7 @@ SECTIONS
         *(.tdata .tdata.* .gnu.linkonce.td.*)
         _tdata_end = . ;
     }
-    .tbss :
+    .tbss (NOLOAD):
```

Comment:
This is already implicitly present in the current toolchain right? 
```
  5 .tbss         0000002c  016c3b90  016c3b90  016b3b8f  2**3
                  ALLOC, THREAD_LOCAL
```

Reply:
Yes that's correct for lld too. I just added NOLOAD here for intentionality and to avoid potential bugs with other linkers and/or older ones.